### PR TITLE
Add inverse matrix

### PR DESCRIPTION
**Pull request by jerivasuaq**
Add inverse matrix

### CODE REVIEW

File: matrix.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def read(self):
                 self.matrix[i][j] = int(input())
 
     def print(self):
-        #print(f"The given matrix is:")
+        # print(f"The given matrix is:")
```
Comment (Fractalemma) [inline matrix.py:22]: This could be removed

File: matrix.py [hunk 2]
```diff
@@ -41,17 +41,58 @@ def mul(self, B):
...
+        adj[2][1] = -(matrix[0][0] * matrix[1][2] - matrix[0][2] * matrix[1][0])
+        adj[2][2] = matrix[0][0] * matrix[1][1] - matrix[0][1] * matrix[1][0]
+
+        return adj
+
+    def inv(self):
+        matrix = self.matrix
+        det = self.det()
+
+        # Si el determinante es 0, la matrix no tiene inversa
```
Comment (Fractalemma) [inline matrix.py:88]: Please remember, comments in English :)

Comment (PinoCrtiaturo) [inline matrix.py:88]: Speak English please sir

File: matrix.py [hunk 3]
```diff
@@ -41,17 +41,58 @@ def mul(self, B):
...
+        if det == 0:
+            print("La matrix no tiene inversa (determinante = 0).")
+            return None
+
+        # Calcular la adjunta
+        adj = self.adj()
+
```
Comment (CaballeroTP) [inline matrix.py:95]: This is a good change, aprroved!

File: matrix.py (hunk 2 above)
Comment (LuisMaga1) [reply matrix.py:88]: I am worry about the matrix transposed, try make the unitary prube, however the rest appear to be ok

**Review by Fractalemma — CHANGES_REQUESTED** [review]
Please add a test where the inverse fails.
The inverse matrix test didn´t work, please double check it

**Review by LuisMaga1 — APPROVED** [review]

File: matrix.py
Comment (PinoCrtiaturo) [inline matrix.py:1]: Add the examples and the answers

**Review by PinoCrtiaturo — APPROVED** [review]

**Review by CaballeroTP — APPROVED** [review]
for the Inverse matriz, you need change error from this method. 

for the all is good.

**Pull request merged**